website/i18n/zh/code.json

```diff
@@ -30,5 +30,221 @@
+  },
+  "features.component.easyDashboard.message": {
+    "message": "Apache APISIX Dashboard 让用户尽可能直观、容易地通过可视化界面来操作 Apache APISIX，它同样是 Apache 软件基金会中持续迭代的开源项目，欢迎你你随时提出新的想法。",
+    "description": "The Apache APISIX Dashboard is designed to make it as easy as possible for users to operate Apache APISIX through a frontend interface. It’s opensource and ever evolving, feel free to contribute."
+  },
+  "features.component.easyDashboard.downloadBtn": {
+    "message": "立即下载",
+    "description": "Downloads"
+  },
+  "features.component.userFlexible.title": {
+    "message": "配置更灵活",
+    "description": "User flexible"
+  },
+  "features.component.userFlexible.message": {
+    "message": "Apache APISIX Dashboard 极大满足了用户需求。不仅提供了丰富的接口，还支持自动创建自定义模块选项，以及无代码流程。",
+    "description": "The Apache APISIX dashboard is flexible to User demand, providing option to create custom modules through code matching your requirements, alongside the existing no-code toolchain."
+  },
+  "features.component.pluginised.title": {
+    "message": "可插拔式工作流",
+    "description": "Pluginised workflow"
+  },
+  "features.component.pluginised.message": {
+    "message": "拒绝重复造轮子。使用 Apache APISIX 内置的插件，可以在极短时间内创建可靠且高性能的系统。你可以像拼乐高一样，将多样插件进行定制化拼接，打造属于你自己的系统。",
+    "description": "No need to reinvent the wheel again and again. Use inbuilt plugins to create high performance systems in tight deadlines. For something custom, there is option of building custom plugins."
+  },
+  "benefits.component.performance.title": {
+    "message": "性能",
+    "description": "Performance"
+  },
+  "benefits.component.performance.subtitle": {
+    "message": "极致的性能体验",
+    "description": "Ultimate performance"
+  },
+  "benefits.component.performance.message": {
+    "message": "Apache APISIX 在后端使用 radixtree-route-matching 和 etcd 为你提供创建高速同步系统的能力。从路由到内置插件，均具有极低延迟的超级性能。",
+    "description": "Apache APISIX uses radixtree-route-matching and etcd under the hood to provide you the ability to create high speed synchronized systems. From routing to built-in plugins, all these are designed and implemented to be uber performant with the minimum latency possible."
+  },
+  "benefits.component.security.title": {
+    "message": "安全",
+    "description": "Security"
+  },
+  "benefits.component.security.subtitle": {
+    "message": "阻拦恶意程序",
+    "description": "Shield against the malicious"
+  },
+  "benefits.component.security.message": {
+    "message": "Apache APISIX 为身份认证和接口验证提供了多个安全插件，将稳定性和安全性放在首位。如需了解更多细节，请查看",
+    "description": "Apache APISIX provides multiple security plugins for identity authentication and interface verification, putting stability and security first. For more information, check"
+  },
+  "benefits.component.security.link.here": {
+    "message": "这里",
+    "description": "here"
+  },
+  "benefits.component.scalability.title": {
+    "message": "高可用与拓展性",
+    "description": "Scalability and availability"
+  },
+  "benefits.component.scalability.subtitle": {
+    "message": "与用户一起扩容",
```

Comment:
```suggestion
    "message": "动态扩容",
```

Reply:
ditto

Reply:
I would suggest keeping the current version :)

Reply:
> I would suggest keeping the current version :)

ok